src/main.rs

```diff
@@ -14,42 +14,59 @@ fn main() -> Result<(), Box<dyn Error>> {
         process::exit(0);
     }
 
-    let warnings = dotenv_linter::run(&args, &current_dir)?;
+    let outputs = dotenv_linter::run(&args, &current_dir)?;
 
-    if warnings.is_empty() {
+    if outputs.is_empty() {
         process::exit(0);
     }
 
-    let total = warnings.len();
+    let total = outputs.iter().map(|o| o.warnings.len()).sum();
     let is_not_quiet = !args.is_present("quiet");
 
     if args.is_present("fix") {
         if is_not_quiet {
-            warnings.iter().for_each(|w| println!("{}", w));
-            println!();
+            dotenv_linter::print_outputs(outputs);
+        } else {
+            outputs.iter().for_each(|w| w.print_backup());
         }
-
-        println!("All warnings are fixed. Total: {}", total);
+        print_fix_total(total);
         process::exit(0);
-    } else {
-        warnings.iter().for_each(|w| println!("{}", w));
+    }
 
-        if is_not_quiet {
-            print_total(total);
-        }
+    if is_not_quiet {
+        dotenv_linter::print_outputs(outputs);
+        print_check_total(total);
+    } else {
+        outputs.iter().for_each(|w| w.print_warnings());
```

Comment:
I would like to have the only one method instead of `if/else` condition. For example:
```rust
let is_quiet = args.is_present("quiet");
output.print_check_output(total, is_quiet);
```

Reply:
We will do it next time.